Makefile

```diff
@@ -18,41 +18,45 @@ virtualenv: $(PYTHON)
 $(PYTHON):
 	$(VIRTUALENV) $(VENV)
 
-install-dev: $(INSTALL_STAMP) $(DEV_STAMP)
+install-dev: $(INSTALL_STAMP) $(DEV_STAMP) ## Install development dependencies
 $(DEV_STAMP): $(PYTHON) dev-requirements.txt
 	$(VENV)/bin/pip install -Ur dev-requirements.txt
 	touch $(DEV_STAMP)
 
 remove-install-stamp:
 	rm $(INSTALL_STAMP)
 
-update: remove-install-stamp install
+update: remove-install-stamp install ## Update the dependencies
 
-serve: install
+serve: install ## Run the ihatemoney server
+	@echo 'Running ihatemoney on http://localhost:5000'
 	$(PYTHON) -m ihatemoney.manage runserver
 
-test: $(DEV_STAMP)
+test: $(DEV_STAMP) ## Run the tests
 	$(VENV)/bin/tox
 
-release: $(DEV_STAMP)
+release: $(DEV_STAMP) ## Release a new version
```

Comment:
Maybe add a link to https://ihatemoney.readthedocs.io/en/latest/contributing.html#how-to-release ?